<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditCoffeeForm</class>
 <widget class="QDialog" name="AddEditCoffeeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить / Редактировать кофе</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>Название сорта:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nameLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="roastLabel">
      <property name="text">
       <string>Степень обжарки:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="roastLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="typeLabel">
      <property name="text">
       <string>Тип:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="typeComboBox">
      <item>
       <property name="text">
        <string>Молотый</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>В зернах</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="descriptionLabel">
      <property name="text">
       <string>Описание вкуса:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTextEdit" name="descriptionTextEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="priceLabel">
      <property name="text">
       <string>Цена:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="priceLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="volumeLabel">
      <property name="text">
       <string>Объем:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="volumeLineEdit"/>
    </item>
    <item row="6" column="0" colspan="2">
     <layout class="QHBoxLayout">
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
</ui>
